Plain Craft Launcher 2/Resources/Language/en_US.xaml

```diff
@@ -2,9 +2,38 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:s="clr-namespace:System;assembly=mscorlib">
 
-    <s:String x:Key="LangTitleOther">More</s:String>
     <s:String x:Key="LangTitleDownload">Download</s:String>
     <s:String x:Key="LangTitleHome">Launch</s:String>
     <s:String x:Key="LangTitleSetup">Setting</s:String>
+    <s:String x:Key="LangTitleLink">Link</s:String>
+    <s:String x:Key="LangTitleOther">More</s:String>
+    <s:String x:Key="LangToolTipBackToTop">Back to top</s:String>
+    <s:String x:Key="LangToolTipDownload">Download Management</s:String>
+    <s:String x:Key="LangToolTipSurrender">Surrender</s:String>
+    <s:String x:Key="LangToolTipCloseMinecraft">Close Minecraft</s:String>
+    <s:String x:Key="LangDialogBtnOK">OK</s:String>
+    <s:String x:Key="LangDialogBtnCancel">Cancel</s:String>
+    <s:String x:Key="LangDialogBtnUpdateFullLog">Full update log</s:String>
+    <s:String x:Key="LangDialogUpdateTo">PCL has been updated to {0}</s:String>
+    <s:String x:Key="LangDialogPolicyContent">Before using PCL, please agree to the PCL user agreement and disclaimer.</s:String>
+    <s:String x:Key="LangDialogPolicyTitle">Agreement authorization</s:String>
```

Comment:
If the location of the term is title, then it should be capital.
`Agreement Authorization`